README.md

```diff
@@ -104,47 +116,44 @@ cd rust
 
 # What is Abacus?
 
-We present Abacus — a system for sending messages between consensus systems
-without paying header validation costs by creating the illusion of cross-chain
-communication. Similar to an atomic swap, Abacus uses non-global protocol
-validation to simulate cross-chain communication. Abacus can carry arbitrary
-messages (raw byte vectors), uses a single-producer multi-consumer model, and
-has protocol overhead sublinear in the number of messages being sent.
+Abacus is a protocol that enables interchain applications by passing messages, any raw bytes, 
+between smart contract blockchains. It does so without bearing the expensive costs involved
+with header verification. Notably, Abacus has economies of scale in messaging costs, and
+brings a new security model to the fore in its Proof of Stake combined with our very own;
+Sovereign Consensus.
+
 
 ## Key Points
 
 System sketch:
 
-1. An "outbox" chain commits messages in a merkle tree
-2. Bonded "validators" attest to the commitment via "checkpoints"
-3. Attested checkpoints are relayed to any number of "inbox" chains
+1. An "outbox" contract on a source chain receives messages from applications and commits messages in a merkle tree
+2. Staking "validators" attest to the commitment via "checkpoints"
+3. Attested checkpoints are relayed to any number of "inbox" contract on any number of destination chains
+4. Messages are processed, not before they are checked against the Sovereign Consensus rules of destination applications.
 
 As a result, one of the following is always true:
 
 1. All inboxes have a valid commitment to messages from the outbox chain
 2. Misbehaving validators can be slashed on the outbox chain
 
-This guarantee, although weaker than header-chain validation, is still likely
-acceptable for most applications.
+While this guarantee may be weaker than header verification in its own right, when augmented with Sovereign Consensus we believe it is more than sufficient for even the most high stakes applications.
 
 ## Summary
 
-Abacus is a new strategy for simulating cross-chain communication without
-validating headers. The goal is to create a single short piece of state (a
-32-byte hash) that can be updated regularly. This hash represents a merkle tree
-containing a set of cross-chain messages being sent by a single chain (the
-"outbox" chain for the Abacus system). Contracts on the outbox chain can submit
-messages, which are put into a merkle tree (the "message tree"). The message
-tree's root may be transferred to any number of "inbox" chains.
+Abacus is a protocol for interchain communication. The protocol's goal is to use generalized messaging between smart contract blockchains to enable interchain applications. 
```

Comment:
I just noticed this now, but "What is abacus" seems to repeated 3 times in this readme, we should probably consolidate